### ISSUE TITLE
Error: No kind "CronJob" is registered for version "batch/v1"

### ISSUE THREAD

**Issue by CermakM** [thoth-station/kebechet#224]
When following the deployment documentation, I get the following error after running the `ansible-playbook` command.

`Error: No kind "CronJob" is registered for version "batch/v1"`

Full ansible task output:

```
{
    "changed": true,
    "cmd": [
        "oc",
        "new-app",
        "--namespace",
        "macermak-thoth-dev",
        "--template=macermak-thoth-dev/kebechet-cronjob"
    ],
    "delta": "0:00:01.052772",
    "end": "2019-07-18 13:00:19.309656",
    "msg": "non-zero return code",
    "rc": 1,
    "start": "2019-07-18 13:00:18.256884",
    "stderr": "    error: no kind \"CronJob\" is registered for version \"batch/v1\" in scheme \"k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:29\"",
    "stderr_lines": [
        "    error: no kind \"CronJob\" is registered for version \"batch/v1\" in scheme \"k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:29\""
    ],
    "stdout": "--> Deploying template \"macermak-thoth-dev/kebechet-cronjob\" for \"macermak-thoth-dev/kebechet-cronjob\" to project macermak-thoth-dev\n\n     Kebechet CronJob\n     ---------\n     This is Kebechet CronJob Config, this template is meant to be used by Bots, but could also be used by humans...\n\n--> Creating resources ...\n--> Failed",
    "stdout_lines": [
        "--> Deploying template \"macermak-thoth-dev/kebechet-cronjob\" for \"macermak-thoth-dev/kebechet-cronjob\" to project macermak-thoth-dev",
        "",
        "     Kebechet CronJob",
        "     ---------",
        "     This is Kebechet CronJob Config, this template is meant to be used by Bots, but could also be used by humans...",
        "",
        "--> Creating resources ...",
        "--> Failed"
    ]
}
```

**Comment by CermakM** [comment]
Related? https://github.com/openshift/origin/issues/21789

**Comment by CermakM** [comment]
Solved by #225. Closing.
